pkgs/data/themes/gruvbox-gtk-theme/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gtk-engine-murrine
```

```suggestion
, gnome-themes-extra
, gtk-engine-murrine
```

It is mentioned in the [README.md](https://github.com/Fausto-Korpsvart/Gruvbox-GTK-Theme#installing-themes) file:

> Before applying the themes you need to check if the Murrine Engine and gnome-themes-extra or gnome-themes-standard packages are already installed on your distro, as it is necessary for the correct rendering of the themes.